docs/google-data-access-tests.md

```diff
@@ -0,0 +1,44 @@
+3. Assertions. Finally, check the validity of the results.
+
+## Google projects and service accounts
+
+### Existing setup
+
+| Google project name      | Owner                   | Reference in the tests              | Purpose |
+|--------------------------|-------------------------|-------------------------------------|---------|
+| dcf-integration          | planx-pla.net           |                                     | Owns buckets dcf-integration-qa and dcf-integration-test |
+| SimpleProjectAlpha       | gen3.autotest@gmail.com | googleProjectA                      | Valid project |
+| ProjectWithComputeAPI    | gen3.autotest@gmail.com | googleProjectWithComputeServiceAcct | Valid project |
+| ProjectWithComputeAPI    | gen3.autotest@gmail.com | googleProjectWithInvalidServiceAcct | Used to test invalid SA registration |
+| planxparentproject       | dummy-one@planx-pla.net | googleProjectWithParentOrg          | Used to test invalid SA registration |
+| projectfencenoaccess     | gen3.autotest@gmail.com | googleProjectFenceNotRegistered     | Used to test invalid SA registration |
+| ProjectServiceAcctHasKey | gen3.autotest@gmail.com | googleProjectServiceAcctHasKey      | Used to test invalid SA registration |
+| Gen3QA-[namespace]       | gen3.autotest@gmail.com | googleProjectDynamic                | See "Notes on googleProjectDynamic" section |
+
+| Service account                                                 | Purpose |
+|-----------------------------------------------------------------|---------|
+| fence-service@dcf-integration.iam.gserviceaccount.com           | Fence monitor. Needs "Editor" and "Project IAM Admin" permissions on the Google projects |
+| gen3qa-service@gen3qa-validationjobtest.iam.gserviceaccount.com | Service account used by the tests to interact with the Google projects. It has access to everything |
+| service-account@gen3qa-[namespace].iam.gserviceaccount.com      | Represents a user who registers a Google project. Each googleProjectDynamic has one |
+
+### Notes on googleProjectDynamic
+
+Most Google projects that have been set up for testing purposes are not modified by the tests.  However, some tests require modifying the Google project itself (service account external key creation/deletion, permission updates...). If several of these tests are running simultaneously, they will make each other fail - for example, if a test generates an external key for a service account, other tests will not be able to register service accounts in the same project.
+
+All the tests that require exclusive access to a Google Project use the same project: `googleProjectDynamic`. Each Jenkins environment has its own "dynamic project" that it has exclusive access to. During the test setup, we make sure to reset this project to what it should be before the tests modify it.
+
+When setting up a new Jenkins environment, we need to set up a new Google project:
```

Comment:
are there docs elsewhere on how to set up a new jenkins env? might want to link to this there if it exists